content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -235,7 +235,7 @@ at different times.
 {{< note >}}
 Clients of the EndpointSlice API must iterate through all the existing EndpointSlices
 associated to a Service and build a complete list of unique network endpoints. It is
-important to mention that endpoints may be duplicated in different EndointSlices.
+important to mention that endpoints may be duplicated in different EndpointSlices.
```

Comment:
This small fix looks good to me.

Deploy Preview: https://deploy-preview-39745--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/endpoint-slices/#duplicate-endpoints